README.md

```diff
@@ -26,6 +26,43 @@ The plugin was born with the purpose of solving some problems of the [eslint-loa
 | Using cache directly from eslint |  :heavy_check_mark:   | :heavy_multiplication_x: |
 | Lint only changed files          |  :heavy_check_mark:   | :heavy_multiplication_x: |
 
+## Migrate from `eslint-loader`
+
+The loader `eslint-loader` will be deprecated soon, please use this plugin instead.
```

Comment:
Can we move it on top of the file? And using `**DEPRECATION**`?

Reply:
`**DEPRECATION**` would be in the `eslint-loader,` right?

Reply:
Here in `eslint-webpack-plugin` it would be just a migration guide

Reply:
Should this migration guide be on the `eslint-loader` only?

Reply:
Oh, yes, my mistake :smile: All fine

> Should this migration guide be on the eslint-loader only?

On both repo for better undestanding